web/src/beta/pages/EditorPage/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC } from "react";
+import { useParams } from "react-router-dom";
+
+import NotFound from "@reearth/beta/components/NotFound";
+import Editor from "@reearth/beta/features/Editor";
+import { isTab } from "@reearth/beta/features/Navbar";
+
+type Props = {};
+
+const EditorPage: FC<Props> = () => {
+  const { sceneId, tab } = useParams<{ sceneId: string; tab: string }>();
+
+  if (!sceneId || !tab || !isTab(tab)) {
+    return <NotFound />;
+  }
+
+  return <Editor tab={tab} sceneId={sceneId} />;
```

Comment:
Can you combine these like below? This component will most likely become large so should keep things as tight and tidy as possible
```
  return (!sceneId || !tab || !isTab(tab)) ? <NotFound /> : <Editor tab={tab} sceneId={sceneId} />;
```

Reply:
(I think we should avoid combining the condition because the condition is a little complex. If we do that, it will be looks bad to me... Also if others would like to add some condition to here, they will keep the combined way. Then It will be too complicated.)

Reply:
I thought a similar thing.
Is it no problem to change when the component gets large if needed?

Reply:
I'd prefer us to keep things simpler and cleaner now, and then refactor if necessary. I don't think a single conditional that has a NotFound as a fallback is "too complicated".

If it gets more complex/large and the conditional gets hard to understand, I think the component needs a little rethinking/refactoring. 

Having a single return to look at where we can see the UI components is much simpler to read than having to search the body of the component for any/all returns...

Reply:
[Fixed!](https://github.com/reearth/reearth/pull/463/commits/8b7470a2039c593fcd01539bf2c9e34d6c59e633)